### PR TITLE
add kbdmini keyboard;dztech and kbdfans keyboards small fix

### PR DESCRIPTION
**Pull request by moyi4681**

### CODE REVIEW

File: keyboards/dz60/config.h [hunk 1]
```diff
@@ -42,5 +42,5 @@
 #define RGBLIGHT_HUE_STEP 8
 #define RGBLIGHT_SAT_STEP 8
 #define RGBLIGHT_VAL_STEP 8
-
+#define RGBLIGHT_SLEEP
```
Comment (drashna) [inline keyboards/dz60/config.h]: Did you mean to add this?

Comment (moyi4681) [reply keyboards/dz60/config.h]: yes:)

File: keyboards/kbdfans/kbd67/mkiirgb/mkiirgb.c [hunk 2]
```diff
@@ -117,3 +117,15 @@ void suspend_wakeup_init_kb(void)
     rgb_matrix_set_suspend_state(false);
     suspend_wakeup_init_user();
 }
+
+__attribute__ ((weak))
+void rgb_matrix_indicators_user(void)
+{
+    if (IS_LED_ON(host_keyboard_leds(), USB_LED_CAPS_LOCK))
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/mkiirgb/mkiirgb.c]:
```suggestion
    if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK))
```

File: keyboards/dztech/dz60rgb/dz60rgb.c [hunk 3]
```diff
@@ -452,3 +109,12 @@ void suspend_wakeup_init_kb(void)
     rgb_matrix_set_suspend_state(false);
     suspend_wakeup_init_user();
 }
+__attribute__ ((weak))
+void rgb_matrix_indicators_user(void)
+{
+    if (IS_LED_ON(host_keyboard_leds(), USB_LED_CAPS_LOCK))
```
Comment (fauxpark) [inline keyboards/dztech/dz60rgb/dz60rgb.c]:
```suggestion
    if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK))
```

File: keyboards/dztech/dz60rgb_ansi/dz60rgb_ansi.c [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+{
+    rgb_matrix_set_suspend_state(false);
+    suspend_wakeup_init_user();
+}
+
+
+__attribute__ ((weak))
+void rgb_matrix_indicators_user(void)
+{
+    if (IS_LED_ON(host_keyboard_leds(), USB_LED_CAPS_LOCK))
```
Comment (fauxpark) [inline keyboards/dztech/dz60rgb_ansi/dz60rgb_ansi.c]:
```suggestion
    if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK))
```

File: keyboards/dztech/dz60rgb_wkl/dz60rgb_wkl.c [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+void suspend_wakeup_init_kb(void)
+{
+    rgb_matrix_set_suspend_state(false);
+    suspend_wakeup_init_user();
+}
+
+__attribute__ ((weak))
+void rgb_matrix_indicators_user(void)
+{
+    if (IS_LED_ON(host_keyboard_leds(), USB_LED_CAPS_LOCK))
```
Comment (fauxpark) [inline keyboards/dztech/dz60rgb_wkl/dz60rgb_wkl.c]:
```suggestion
    if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK))
```

File: keyboards/dztech/dz65rgb/dz65rgb.c [hunk 6]
```diff
@@ -120,6 +124,6 @@ void rgb_matrix_indicators_user(void)
 {
     if (IS_LED_ON(host_keyboard_leds(), USB_LED_CAPS_LOCK))
```
Comment (fauxpark) [inline keyboards/dztech/dz65rgb/dz65rgb.c]:
```suggestion
    if (IS_HOST_LED_ON(USB_LED_CAPS_LOCK))
```

**Comment by moyi4681** [comment]
@fauxpark pls check and confirm ,thanks:)

**Comment by fauxpark** [comment]
Why was the dz40rgb folder deleted? Does it not actually exist?

**Comment by moyi4681** [comment]
> Why was the dz40rgb folder deleted? Does it not actually exist?
> Otherwise, LGTM.

No more exist

**Comment by moyi4681** [comment]
when will this pull be merged? kbdmini need to be compile in config.qmk.fm, pls check and confirm, thank u very much:) @drashna @fauxpark

**Comment by drashna** [comment]
@moyi4681 To be blunt here, this is a big PR with a lot of changes.  It ... isn't exactly simple to go through, due to all the changes, and the scope of the PR.

In the future, it may make things easier if you break this into distinct parts.  Like breaking up the dz60rgb changes into one PR, adding the kbdmini into another PR, etc.   This way, each PR is small, fairly self contained, and more importantly, easier to review.   The easier it is on us, the more likely that it will be quickly reviewed and quickly merged.   Especially if experience is important.

That said, looks good!  And thank you for the contribution!

**Pull request merged**